package.json

```
@@ -79,8 +79,9 @@
 		"@discordjs/rest": "^2.0.1",
 		"@discordjs/voice": "^0.18.0",
 		"@distube/yt-dlp": "^2.0.1",
+		"@distube/ytdl-core": "^4.16.6",
```

Biome (1.9.4)

[error] The key @distube/ytdl-core was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)